lib/FusionInventory/Agent/Task/Inventory/MacOS/CPU.pm

```diff
@@ -53,7 +53,8 @@ sub _getCpus {
     }
     close $handle;
 
-    my $type  = $sysprofile_info->{'Processor Name'} ||
+    my $type  = $sysctl_info->{'machdep.cpu.brand_string'} || 
```

Comment:
too late, the trimWithespace has been added and PR merged yesterday, to be in the 2.3.20 release :)